ios/FluentUI/PreviewCard/PreviewCard.swift

```diff
@@ -28,38 +28,64 @@ import SwiftUI
     private var previewCardView: PreviewCard!
 }
 
+extension View {
+    /// Sets the height and width of the Preview Card.
+    func cardBorderModifiers() -> some View {
```

Comment:
These modifiers are being called just once in the implementation. I believe that just moving their implementation to where they're called is better for now.
If you need to apply the same functionality in multiple places, this is a good approach.